apps/i18n/common/en_us.json

```diff
@@ -410,6 +410,14 @@
   "childAccountConsentValidPermission": "Permission:",
   "childAccountConsentValidPermissionGranted": "Granted on {date}",
   "childAccountConsentValidMessage": "Thank you for approving your child’s Code.org account! You will receive a confirmation email within 24–48 hours. Please let your child know you’ve approved their account so they can get started learning and coding today!",
+  "childAccountPolicy_consentStatus": "Consent Status",
+  "childAccountPolicy_consentProcessReadMore": "Read more about this [consent process and these changes here]({url}).",
+  "childAccountPolicy_lockedOut": "Locked Out",
+  "childAccountPolicy_studentParentalConsentStatus": "Student Parental Consent Status",
+  "childAccountPolicy_studentParentalConsentNotice": "The following students will require parental consent by 7/1/2024 to continue using Code.org with their personal account. Please ensure these students provide a parent or guardian email address to complete the parental consent process.",
```

Comment:
Normally the "7/1/2024" should be a dynamic value pulled from the CPA configuration, but I think this string will change soon due to the Grace Period feature.